scripts/find_pulsar_in_obs.py

```diff
@@ -192,6 +192,14 @@ def grab_source_alog(source_type='Pulsar', pulsar_list=None, max_dm=1000., inclu
                 else:
                     name_ra_dec.append([line[0], line[1], line[2]])
 
+    elif source_type == "POI":
+        #POI = points of interest
+        db = open(os.environ["POI_CSV"], "r")
+        for line in db.readlines():
+            if not line.startswith("#"):
+                line = line.split(",")
+                name_ra_dec.append([line[0], line[1], line[2]])
+     
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4912493)